CodeGeneration/Sources/generate-swiftsyntax/templates/swiftparser/ParserEntryFile.swift

```diff
@@ -101,5 +101,23 @@ let parserEntryFile = SourceFileSyntax(leadingTrivia: copyrightHeader) {
       }
       """
     )
+
+    DeclSyntax(
+      """
+      mutating func parseNonOptionalCodeBlockItem() -> RawCodeBlockItemSyntax {
+        guard let node = self.parseCodeBlockItem() else {
+          // The missing item is not neccessary to be a declaration,
+          // which is just a placeholder here
+          let invalidCodeBlockItemNode = RawCodeBlockItemSyntax(
+            item: .decl(RawDeclSyntax(RawMissingDeclSyntax(attributes: nil, modifiers: nil,arena: self.arena))),
+            semicolon: nil,
+            arena: self.arena
+          )
+          return invalidCodeBlockItemNode
```

Comment:
There’s no real need for the variable here

```suggestion
          return RawCodeBlockItemSyntax(
            item: .decl(RawDeclSyntax(RawMissingDeclSyntax(attributes: nil, modifiers: nil,arena: self.arena))),
            semicolon: nil,
            arena: self.arena
          )
```